replication-api/src/main/java/com/connexta/replication/api/data/Site.java

```diff
@@ -44,16 +37,9 @@
    *
    * @param url the URL to give this site
    */
+  // still internally used by the ReplicatorImpl
```

Comment:
If we want to get rid of these now, we can just copy the object with the updated values and re-save it with the same ID and the original record will get overwritten.

Reply:
Didn't want to do that as part of this PR especially since some of that might/will change as part of our redesign. I just cleaned up what I could for now and left the rest such that it wouldn't break things for now.